workflows/diagnostics/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -332,6 +332,25 @@ def zonal_mean_bias_hovmoller(diag_arg: DiagArg):
     return zonal_means
 
 
+@registry_2d.register("deep_tropical_meridional_mean_value")
+@transform.apply(transform.resample_time, "3H", inner_join=True)
+@transform.apply(transform.daily_mean, datetime.timedelta(days=10))
+@transform.apply(transform.subset_variables, ["total_precip_to_surface", "ULWRFtoa"])
+def deep_tropical_mean_hovmoller(diag_arg: DiagArg):
+    logger.info(f"Preparing deep tropical meridional mean values (2d)")
+    prognostic, grid = diag_arg.prediction, diag_arg.grid
+    deep_tropical_means = xr.Dataset()
+    for var in prognostic.data_vars:
+        logger.info(f"Computing deep tropical meridional mean (2d) over time for {var}")
+        with xr.set_options(keep_attrs=True):
+            deep_tropical_means[var] = vcm.meridional_average_approximate(
+                grid.lon,
+                prognostic[[var]].where(np.abs(grid.lat) <= 15),
```

Comment:
Seems like it should be area weighted. Although it won't matter much in the deep tropics...it could make a big difference at higher latitudes. Is it possible to add a weights option to `vcm.meridional_average_approximate` so you can pass `weights = area.where(np.abs(grid.lat) <= 15)`?

Reply:
I think it should be just as valid as the zonal average function, since this is grouping cells on the cubed sphere, though arguably both should have an option to be area-weighted.  I can add that.

Reply:
Good point. Our current zonal averages weights the tile interiors too much. "approximate" indeed.

The cos(lat) factor could matter more in the meridional case, since the number of grid points in a given latitude bin will  scale with cos(lat).